### PR TITLE
fix(url): base, homepage, bug url updated for all packages in package.json

### LINKED ISSUE #824: URIs in package.json files should be updated for directories under packages
After migrating all common fxa repositories to this monorepo, the following things should be updated in the package.json for all the old repositories like fxa-content-server, fxa-profile-server. This is the [link](https://github.com/mozilla/fxa/blob/7926e11e42030a46ec2e7389f97d0c130e6b4155/packages/fxa-content-server/package.json#L27-L30) for only [fxa-content-server](https://github.com/mozilla/fxa/tree/master/packages/fxa-content-server) package but it should be updated in all packages which have a package.json file.

### PR DESCRIPTION
**Pull request by farhan787**
fixes #824

### CODE REVIEW

File: packages/123done/package.json [hunk 1]
```diff
@@ -13,10 +13,10 @@
     }
   ],
   "homepage": "http://123done.org/",
-  "bugs": "https://github.com/mozilla/123done/issues",
+  "bugs": "https://github.com/mozilla/fxa/issues",
   "repository": {
     "type": "git",
-    "url": "https://github.com/mozilla/123done"
+    "url": "https://github.com/mozilla/fxa/tree/master/packages/123done"
```
Comment (vbudhram) [inline packages/123done/package.json]: I'm not sure if the repository url should be point here. My understanding was that it should point to the git repo that the project belongs too. In this case `https://github.com/mozilla/fxa.git`.

@shane-tomlinson @philbooth Thoughts?

Comment (philbooth) [reply packages/123done/package.json]: Yep, according to the docs it should be the `.git` URL:

https://docs.npmjs.com/files/package.json#repository

Comment (philbooth) [reply packages/123done/package.json]: Or actually, the shorthand `mozilla/fxa` is allowed to, I notice now.

Comment (farhan787) [reply packages/123done/package.json]: @vbudhram @philbooth thanks for correcting, I had that confusion in my mind but I thought it should point to the package but now it's better to point the main repo.

**Comment by farhan787** [comment]
@vbudhram is it good now ?

**Pull request merged**